includes/import/abstract-wc-product-importer.php

```diff
@@ -434,6 +434,11 @@ protected function set_variation_data( &$variation, $data ) {
 			return new WP_Error( 'woocommerce_product_importer_missing_variation_parent_id', __( 'Variation cannot be imported: Missing parent ID or parent does not exist yet.', 'woocommerce' ), array( 'status' => 401 ) );
 		}
 
+		// Stop if parent is a product variation.
+		if ( $parent->post_type === 'product_variation' ) {
+			return new WP_Error( 'woocommerce_product_importer_parent_set_as_variation', __( 'Variation cannot be imported: Parent cannot be a product variation', 'woocommerce' ), array( 'status' => 401 ) );
```

Comment:
Will probably should better like:

```
Variation cannot be imported: Parent product cannot be a variation.
```

Reply:
Awesome! Thanks for the feedback. I just pushed those changes.